File: Perl/built-in/built_in_function.pl
sub reverse_num
{
	my $num = shift;
	my @arr = split('', $num);
	my @rev_arr = reverse(@arr);
	my $return_str = join('', @rev_arr);
	return $return_str
}

print "Please insert a number: ";
$number = <STDIN>;
$res = reverse_num($number);
print $res;

File: TypeScript/built-in/built_in_function.ts
const readline = require('readline')

class Reverse {
	num : string

	constructor(num : string) {
		this.num = num
	}

	reverseNumber() : string {
		return this.num.split("").reverse().join("")
	}
}

const rl = readline.createInterface({
	input: process.stdin,
	output: process.stdout
})

rl.question('Please enter a number: ', (answer) => {
	let rev = new Reverse(answer)
	let res = rev.reverseNumber()
	console.log(res)

	rl.close()
})